### ISSUE TITLE
Tooltip in generate object window

### ISSUE THREAD

**Issue by amra** [Le0Michine/FusionGridfinityGenerator#37]
There's great wiki documentation for the feature: https://github.com/Le0Michine/FusionGridfinityGenerator/wiki/Bin-generator-options

This should be part of the Fusion 360 plugin, where user doesn't have to go to a site and search for the information.

The form should be a tooltip or an `?` icon with a popup.
![image](https://github.com/Le0Michine/FusionGridfinityGenerator/assets/174620/2810ec0b-1210-4729-bc0f-d80d716212ca)

**Comment by Le0Michine** [comment]
I agree that UI needs to be more informative, especially with the amount of parameters it has now.
Unfortunately, to my knowledge, the suggested form of tooltips isn't something supported by UI components available in Fusion. There is support for tooltip text which I might be able to add at some point, however, because interaction isn't that obvious I'm not sure if that would actually be discoverable enough. Currently there is no indication on the component whether it has a tooltip or not and user would need to hover cursor above input field for a second to get it displayed

There is an alternative I saw used by some other apps: usage of images and web views, like this one 
<img width="244" alt="image" src="https://github.com/Le0Michine/FusionGridfinityGenerator/assets/2707008/8b3da9bc-7047-4be0-a23c-dbc9f0498191">

which has some downsides as well due to its size, but it might make sense to explore that as well, especially for things like custom compartments
